app/controllers/events/view/tickets/attendees/list.js

```diff
@@ -13,8 +13,8 @@ export default Controller.extend({
       title        : 'Ticket Name'
     },
     {
-      propertyName : 'ticket.price',
-      title        : 'Ticket Price',
+      propertyName : 'total.price',
```

Comment:
what's total?

Reply:
@CosmicCoder96 Since the table shows total price.

Reply:
@mohitm15 I know that I am asking what's the `total` in the propertyName variable.

Reply:
@CosmicCoder96 I just changed the variable name